src/imports/Components/1.3/Captions.qml

```diff
@@ -124,22 +124,16 @@ ColumnLayout {
     }
 
     Label {
+        Layout.fillWidth: true
```

Comment:
Is this an API break?

Reply:
No. It's using the correct property as anchors cannot be used on elements inside a `*Layout` component.